segment/analytics/client.py

```diff
@@ -64,9 +78,18 @@ def __init__(self,
         self.gzip = gzip
         self.timeout = timeout
         self.proxies = proxies
+        if(oauth_client_id and oauth_client_key and oauth_key_id):
+            self.oauth_manager = OauthManager(oauth_client_id, oauth_client_key, oauth_key_id,
```

Comment:
Don't know if this is strictly necessary, but I was running into `oauth_manager does not exist` errors when running the unit tests (since they don't have oauth settings). I got around that by setting `self.oauth_manager = None` just before this if statement.

Reply:
Fixed